packages/wp-babel-makepot/presets/default.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
 	presets: [ '@babel/preset-react', '@babel/preset-typescript' ],
-	plugins: [ '@babel/plugin-proposal-class-properties' ],
+	plugins: [ '@babel/plugin-proposal-class-properties', '@babel/plugin-transform-private-methods' ],
```

Comment:
The `@babel/plugin-proposal-class-properties` plugin is now deprecated. I suppose because it's not really a "proposal" anymore. If you want you could replace it with `@babel/plugin-proposal-class-properties`.
Or, maybe we just want to include all the approved standardised features by default? That's doable with the `preset-env` preset.

```js
module.exports = {
	presets: [ '@babel/preset-react', '@babel/preset-typescript', '@babel/preset-env' ],
	plugins: [],
};
```